ELISA Policies and Guidelines

```diff
@@ -0,0 +1,69 @@
+
+**ELISA deliverables:**
+- ELISA develops an example qualitative analysis for safety critical use-cases for the Linux kernel e.g: Automotive and Medical.
+- ELISA provides resources for System integrators to apply and use to analyze qualitatively and quantitatively on their systems. For example, ELISA will analyze published CWEs to identify hazards for the two use-cases.
+- System Integrators use ELISA deliverables to analyze their systems
+
+**What are the publications:**
+- Whitepapers and blogs published on elisatech etc.
+- Link to the github repository
+
+Presentations and work in progress are not considered a publication until reviewed and added to one of the publication sites. All work in progress on github should contain licenses.
+
+### Publication Licensing Policy
+All ELISA documentation deliverables shall be released under CC-BY-4.0. All published documents shall carry the following licensing information.
+
+### Licensing Header
+
+**SPDX-License-Identifier: CC-BY-4.0**
+This document is released under the Creative Commons Attribution 4.0 International License, available at https://creativecommons.org/licenses/by/4.0/legalcode. Pursuant to Section 5 of the license, please note that the following disclaimers apply (capitalized terms have the meanings set forth in the license).
+
+To the extent possible, the Licensor offers the Licensed Material as-is and as-available, and makes no representations or warranties of any kind concerning the Licensed Material, whether express, implied, statutory, or other. This includes, without limitation, warranties of title, merchantability, fitness for a particular purpose, non-infringement, absence of latent or other defects, accuracy, or the presence or absence of errors, whether or not known or discoverable.  Where disclaimers of warranties are not allowed in full or in part, this disclaimer may not apply to You.
+
+To the extent possible, in no event will the Licensor be liable to You on any legal theory (including, without limitation, negligence) or otherwise for any direct, special, indirect, incidental, consequential, punitive, exemplary, or other losses, costs, expenses, or damages arising out of this Public License or use of the Licensed Material, even if the Licensor has been advised of the possibility of such losses, costs, expenses, or damages. Where a limitation of liability is not allowed in full or in part, this limitation may not apply to You.
+
+The disclaimer of warranties and limitation of liability provided above shall be interpreted in a manner that, to the extent possible, most closely approximates an absolute disclaimer and waiver of all liability
+
+References to documents that use this header:
+- https://elisa.tech/wp-content/uploads/sites/75/2021/05/ELISA_Advancing-Open-Source-Safety-Critical-Systems_Whitepaper_050521.pdf
+- https://github.com/elisa-tech/wg-medical-devices/commit/79290c454ab8f4f03560090843f84d13ed7b3360
```

Comment:
If I follow the link to medical I get a headline from github mentioning: "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository." I guess we may have to change it to: https://github.com/elisa-tech/wg-medical-devices/commit/e7eefff32dd2e32d4486673e21b8287877ba7a8d